packages/rio/charts/Chart.yaml

```diff
@@ -1,14 +1,13 @@
 apiVersion: v1
 name: rio
 version: 0.8.000
```

Comment:
Do we not care about bumping version since it never went live ?

Reply:
Yes, It is always hidden so overriding it should be fine.